CMakeLists.txt

```diff
@@ -106,33 +106,33 @@ set_target_properties(${PROJECT_NAME}_filter PROPERTIES COMPILE_FLAGS "${CMAKE_C
 set_target_properties(${PROJECT_NAME}_filter PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
 
 # Demo filter executable
-add_executable(${PROJECT_NAME}_filter_test src/demo/grasp_filter_test.cpp)
-target_link_libraries(${PROJECT_NAME}_filter_test
+add_executable(${PROJECT_NAME}_grasp_filter_demo src/demo/grasp_filter_demo.cpp)
+target_link_libraries(${PROJECT_NAME}_grasp_filter_demo
   ${PROJECT_NAME} ${PROJECT_NAME}_filter ${catkin_LIBRARIES} ${Boost_LIBRARIES}
 )
 
 # Demo grasp executable
-add_executable(${PROJECT_NAME}_test src/demo/grasp_generator_demo.cpp)
-target_link_libraries(${PROJECT_NAME}_test
+add_executable(${PROJECT_NAME}_grasp_generator_demo src/demo/grasp_generator_demo.cpp)
+target_link_libraries(${PROJECT_NAME}_grasp_generator_demo
```

Comment:
split this into a test also, where you remove all the visualization aspects and just check that X number of grasps were generated

Reply:
see reply regarding tests